monitoring/roles/elasticsearch/files/values.yaml

```diff
@@ -20,4 +23,4 @@ volumeClaimTemplate:
   storageClassName: do-block-storage
   resources:
     requests:
-      size: 5Gi
+      size: 35Gi
```

Comment:
How did you come up with 35Gi?

Reply:
Ah they were actually already set to 20 or 30 in dev, I just wanted them to reflect reality

I also added ~5 gb on top since they were close